dask_sql/physical/rex/core/call.py

```diff
@@ -613,6 +614,37 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        # TODO: format timestamps for GPU tests
+        if "cudf" in str(type(df)):
+            if format != "%Y-%m-%d %H:%M:%S":
+                print("Formatting timestamps not supported on GPU")
+            if df.dtype == "object":
+                return df
+            else:
+                return df * 10**9
```

Comment:
which case is this handling?

Reply:
This case handles a cuDF Series of integers - we multiply by 10**9 to convert from nanoseconds to seconds. It's essentially the same functionality as when we use `dd.to_datetime(df, unit="s")` for Pandas Series, except that `dd.to_datetime()` doesn't work with cuDF Series.